src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/sizes/lambdabodylength/Example1.java

```diff
@@ -0,0 +1,45 @@
+/*
+LambdaBodyLength
+
+
+*/
+
+package com.puppycrawl.tools.checkstyle.checks.sizes.lambdabodylength;
+
+class Example1 {
+  Runnable r = () -> { // violation 'length is 11 .* (max allowed is 10).'
```

Comment:
Seems reasonable to move "good" lambda to be `r`, and place `ok` comment with explanation there.

Reply:
We can skip `.* (max allowed is 10).` as example CI fig is above all is clear and short.

Better to not use regexp in Examples.

Reply:
My request above sounds like a regexp check idea for Examples. Is is doable in simple way ?

Reply:
> Seems reasonable to move "good" lambda to be r, and place ok comment with explanation there.

> We can skip .* (max allowed is 10). as example CI fig is above all is clear and short.

Done.

> My request above sounds like a regexp check idea for Examples. Is is doable in simple way?

A regexp to check for regex expressions in the `// violation` comments? We have to be very specific - for example escapes `\.`, `\+`, `\(`, `\)`, `\[`, `\]` or common expressions `.*`

Reply:
>common expressions .*

Let's target this most common mistake. We will see if we can add other also later on.

Reply:
> Let's target this most common mistake. We will see if we can add other also later on.

https://github.com/checkstyle/checkstyle/issues/13244 is where discussion continues on this.